# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1466574425 14400
#      Wed Jun 22 01:47:05 2016 -0400
# Node ID 9adb3537848885a498421200c86a2030f0e4a52d
# Parent  b3d19ade4cd50308e31ef60e0daf0d906307282b
* libinterp/module.mk (BUILT_SOURCES): Include oct-tex-parser.h in the list.

diff --git a/libinterp/module.mk b/libinterp/module.mk
--- a/libinterp/module.mk
+++ b/libinterp/module.mk
@@ -30,16 +30,17 @@ libinterp_liboctinterp_la_CXXFLAGS = $(A
 octlib_LTLIBRARIES += libinterp/liboctinterp.la
 
 BUILT_SOURCES += \
   libinterp/builtin-defun-decls.h \
   libinterp/corefcn/defaults.h \
   libinterp/corefcn/graphics-props.cc \
   libinterp/corefcn/graphics.h \
   libinterp/corefcn/mxarray.h \
+  libinterp/corefcn/oct-tex-parser.h \
   libinterp/parse-tree/oct-gperf.h \
   libinterp/parse-tree/oct-parse.h \
   libinterp/version.h
 
 ULT_PARSER_SRC := \
   libinterp/corefcn/oct-tex-lexer.in.ll \
   libinterp/corefcn/oct-tex-parser.in.yy \
   libinterp/parse-tree/oct-parse.in.yy
